samples: nrf9160: modem_shell: stored commands could be set to be run after a startup (#8077)
nrfconnect/sdk-nrf: doc/nrf/releases/release-notes-changelog.rst

      When running this configuration, you can perform BT scanning and advertising using the ``bt`` command.
    * Support for injecting GNSS reference altitude for low accuracy mode.
      For a position fix using only three satellites, GNSS module must have a reference altitude that can now be injected using the ``gnss agps ref_altitude`` command.

>     * New command ``startup_cmd`` which can be used to store up to 3 MoSh commands to be run after a start/bootup.

Author: OK, I'll add example to Readme and remove example from help text.

>     * New command ``startup_cmd`` which can be used to store up to 3 MoSh commands to be run after a start/bootup.

Author: Example added. Please, check.

>     * New command ``startup_cmd`` which can be used to store up to 3 MoSh commands to be run after a start/bootup.

Reviewer: ```suggestion
    * New command ``startup_cmd``, which can be used to store up to three MoSh commands to run on start/bootup.
```

>       As a default, commands are run after a default PDN context is getting active, but optionally can be set to be run N seconds after a bootup.

Reviewer: ```suggestion
      By default, commands are run after the default PDN context is activated, but can be set to run N seconds after bootup.
```
